test/parallel/test-cluster-master-kill.js

```diff
@@ -81,7 +81,8 @@ if (cluster.isWorker) {
   process.once('exit', () => {
     assert.strictEqual(typeof pid, 'number',
                        `got ${pid} instead of a worker pid`);
-    assert.strictEqual(alive, false, 'worker was alive after master died');
+    assert.strictEqual(alive, false, 
+                       `${alive} worker was alive after master died`);
```

Comment:
I could be misunderstanding something but wouldn't this just be `true worker was alive after master died`? That said I'm not certain what the request was at the Code & Learn.

Reply:
probably the intent is to print the actual value of `alive` to aid diagnostics, but in this case looks like it is redundant - as the value is evident (`true`, as equating to `false` was resulted in `false`) and can be inferred statically.

Reply:
Theoretically, at least, `isAlive()` could return any value. Printing it here may be a bit much, though. I guess it could be:

```js
assert.strictEqual(alive, false, `worker was alive after master died (alive = ${alive})`);
```

I'd be :+1: on that change.